src/main/resources/site/mixins/accordion/accordion.xml

```diff
@@ -0,0 +1,13 @@
+<mixin>
+    <display-name>Trekkspill</display-name>
+    <form>
+        <input name="accordion" type="ContentSelector">
+            <label>Trekkspill-innhold</label>
+            <occurrences minimum="1" maximum="99"/>
```

Comment:
No need to set max 99, min 1 max 0 will require 1, but you'll be able to set an endless amount of accordions. No one will most likely ever reach 99, but still no reason to limit it